content/pages/community/events.md

```diff
@@ -25,31 +25,20 @@ Sidebar_menu_elem_url_6: community/sponsorship
 Content_layout: multiple-columns
 ---
 
+__See something missing?__ Events come and go at a speed that is sometimes hard to keep track of. Do you know of an event not listed here? Did you spot out-of-date information? Can you otherwise improve on this page? Plaese, [submit a pull request!](https://github.com/xsf/xmpp.org/edit/master/content/pages/community/events.md)
+
 ## XMPP Summits
 
 The XSF organizes periodic in-person gatherings in order to coordinate high-bandwidth conversations, interop testing, brainstorming and planning, and to conduct other XSF business face-to-face.
```

Comment:
Maybe mention here when the last summit was. This makes it clear to the reader that it is not strange that the next summit isn’t announced yet (because it was literally a few weeks ago).

Ideally, we had a blog post with a review of the summit to link to :-).